docs/getting_started.md

```diff
@@ -21,7 +21,8 @@ chmod +x /usr/local/bin/argocd
 kubectl create namespace argocd
 kubectl apply -n argocd -f https://github.com/argoproj/argo-cd/blob/master/manifests/install.yaml
 ```
-This will create a new namespace, `argocd`; where ArgoCD services and application resources will live.
+This will create a new namespace, `argocd` where ArgoCD services and application resources will live.
+This will also create key replicasets, deployments and services needed for ArgoCD
```

Comment:
> This will also create key replicasets, deployments and services needed for ArgoCD

I think this statement is be confusing: we don't explicitly create replicasets since they are implicitly created by our deployments. Also the fact that we create a deployment/service should also be obvious.